packages/babel-plugin-polyfill-corejs3/README.md

```diff
@@ -26,3 +26,53 @@ Add this plugin to your Babel configuration:
 
 This package supports the `usage-pure`, `usage-global`, and `entry-global` methods.
 When `entry-global` is used, it replaces imports to `core-js`.
+
+## Options
+
+See [here](../../docs/usage.md#options) for a list of options supported by every polyfill provider.
+
+### `version`
+
+`string`, defaults to `"3.0"`.
```

Comment:
Technically it can also be a number `3`. But I lean towards not documenting it at all in favor of the string approach.

Reply:
I wanna disallow numbers here in the future since usage just `corejs: 3` when should be specified a used minor version is one of the most popular mistakes.